frontend/src/AlertsManagement/AlertsContent.js

```diff
@@ -0,0 +1,143 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Box from '@material-ui/core/Box';
+import Paper from '@material-ui/core/Paper';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+import Typography from '@material-ui/core/Typography';
+import AllInboxIcon from '@material-ui/icons/AllInbox';
+import DoneIcon from '@material-ui/icons/Done';
+import ErrorIcon from '@material-ui/icons/Error';
+import AlertsList from './AlertsList';
+
+function TabPanel(props) {
+  const { children, value, index, ...other } = props;
+
+  return (
+    <div
+      role="tabpanel"
+      hidden={value !== index}
+      id={`tabpanel-${index}`}
+      aria-labelledby={`tab-${index}`}
+      {...other}
+    >
+      {value === index && (
+        <Box p={3}>
+          <Typography>{children}</Typography>
+        </Box>
+      )}
+    </div>
+  );
+}
+
+TabPanel.propTypes = {
+  children: PropTypes.node,
+  index: PropTypes.any.isRequired,
+  value: PropTypes.any.isRequired,
+};
+
+let a11yProps = (index) => {
+  return {
+    id: `tab-${index}`,
+    'aria-controls': `tabpanel-${index}`,
+  };
+}
+
+// TODO: fetch alerts from backend and save in arrays.
+const allAlerts = [0, 1, 2, 3, 4, 5, 6];
+const unresolvedAlerts = [0, 1, 2, 3];
+const resolvedAlerts = [4, 5, 6]
```

Comment:
I wrote too much for this comment :/ let me know if you have any other questions about this!

Yeah I probably should've explained the hard coding of these arrays and what I plan to do with the actual alerts. 
I'm thinking of fetching the alerts and saving those in an array (which will be allAlerts). This won't be changed so the indices should be fixed. I plan to make an Alert component which will just put the information of an alert (like the name, date, # of anomalies) into HTML elements and save these in the array, or save their ids in the array while having another structure (like a map between the IDs and the component). Not totally sure if I can do this yet though.
Before putting the alerts in the tabs, I will separate them by unresolved and resolved status, giving two arrays of indices (which refer to where the alerts are in allAlerts), and these two arrays of indices are used to initialize the unchecked and checked arrays in the state. These 2 initial arrays are specifically for initializing the state arrays, so I will probably try to do it in the constructor without keeping actual arrays.
The unchecked and checked arrays will just contain the indices of the respective alerts so the information will be retrieved from the array or the map (whatever ends up being used to store the alerts).

Reply:
Discussed offline.

```
const allAlerts = [id1, id2, id3] where id'x' is the id of AlertX
const alertsMap = {id1: <Alert1>, 2: <Alert2>, 3: <Alert3>, ...};
const unresolvedAlerts=[0, 1]
const resolvedAlerts=[2]
```

the alert id will be the same one stored in the datastore, to be able to tell the backend which alert status got changed.

so maybe in this case instead of referring them as "indices", might be more specific (and less ambiguious) to refer them to "id's"? indices have the connotation of array indexing, which from your description might not be the case.